package.json

```diff
@@ -30,17 +30,12 @@
     "headless.js"
   ],
   "scripts": {
-    "start-webpack": "(cd examples/main && yarn && npm run start-local)",
-    "start": "npm run build-watch && budo dist/bundle.js:dist/lumagl.js --live --open --port 3000 --watch-glob '**/*.{html,css,js,glsl}' ",
-    "build-watch": "babel src -d dist --source-maps inline --copy-files --watch & babel examples -d dist --source-maps inline --copy-files --watch &",
+    "start": "cat < docs/get-started/examples.md",
```

Comment:
> Ah, I forgot there's no "layer browser" equivalent in luma.

I am starting to think this might be the best setup for all our repos (i.e also deck.gl/react-map-gl).

* We see from issues being opened that many users are confused by the double install setup in the current `start` script (I think we knew this). 
* Also users don't seem to easily figure out how to work with the standalone examples, including how to build against local source.
So the idea was, why not just tell them to go into the examples and install and run there.